main.go

```diff
@@ -63,6 +64,9 @@ func optionsFromFile() *client.Options {
 		opt.TagFilters = parseFilterString(cfg.Section("options").Key("tags").Value())
 		opt.Fields = parseFieldsString(cfg.Section("options").Key("fields").Value())
 		opt.Credentials = cfg.Section("options").Key("creds").Value()
+		if parsed, err := strconv.ParseBool(cfg.Section("options").Key("hideHeader").Value()); err == nil {
+			opt.HideHeader = parsed
+		}
```

Comment:
It seems to be a little complecated... 
How about do this? 

```go
opt.HideHeader = (cfg.Section("options").Key("hideHeader").Value() == "true")
```

This allows only `true` chars. And, you don't need to use `strconv` package.